### PR TITLE
feat(obstacle_cruise_planner): clean parameters for optimization based cruise planner

### PR DESCRIPTION
**Pull request by purewater0901**
## Description
Remove unnecessary parameters and redundant parameters.

## Tests performed
PSIm

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by takayuki5168** [comment]
@purewater0901 This is draft PR, but do you have something more to fix?

File: planning/obstacle_cruise_planner/config/obstacle_cruise_planner.param.yaml [hunk 1]
```diff
@@ -10,7 +10,6 @@
       min_ego_accel_for_rss: -1.0 # ego's acceleration to calculate RSS distance [m/ss]
```
Comment (takayuki5168) [inline planning/obstacle_cruise_planner/config/obstacle_cruise_planner.param.yaml:10]: Please apply the same modification to tier4_planning_launch in autoware.universe

**Comment by purewater0901** [comment]
@takayuki5168 Yes, there are still some codes I need to fix. I'll open this PR after I resolved them.

File: planning/obstacle_cruise_planner/config/obstacle_cruise_planner.param.yaml (hunk 1 above)
Comment (purewater0901) [reply planning/obstacle_cruise_planner/config/obstacle_cruise_planner.param.yaml:10]: done

**Review by takayuki5168 — APPROVED** [review]

**Pull request merged**